pkg/network/setup/netconf_test.go

```diff
@@ -49,39 +49,56 @@ var _ = Describe("netconf", func() {
 	})
 
 	It("runs setup successfully", func() {
```

Comment:
I would spit into two sub contexts -
"runs setup successfully"
    ... (all the regular setup tests)
    "with completion cache"

Reply:
I think it is preferable to look at this from the various possible scenarios, not logically like a table of options.
Some tests here require the cache and make no sense without it (e.g. `Teardown`tests, skipping secondary setup runs, etc).

I also like less the indentations of tests.

Maybe I should name the tests better to reflect the existence of cache or not?

Reply:
For me it was confusing that the tests are mixed. Some use the `WithCompletionCache` and some don't.

Reply:
Ok, I placed the tests that uses the cache into a context as you suggested.
Done.